test/e2e/framework/internal/unittests/cleanup/cleanup_test.go

```diff
@@ -128,15 +128,15 @@ INFO: after #1
 [AfterEach] e2e
   cleanup_test.go:76
 INFO: after #2
-[DeferCleanup] e2e
+[DeferCleanup (Each)] e2e
   cleanup_test.go:85
 INFO: cleanup first
-[DeferCleanup] e2e
+[DeferCleanup (Each)] e2e
```

Comment:
@onsi: what does the `(Each)` suffix refer to here?

My initial though was that the `DeferCleanup` must have been called by an `BeforeEach`, but the "cleanup first" comes from `It`.

Reply:
Makes sense now that you explained it.

@dims: I think this can be merged.